src/InsightContainer/index.js

```diff
@@ -0,0 +1,200 @@
+        <Grid item>
+          <TypographyLoader
+            loading={loading}
+            loader={{
+              primaryOpacity: 0.5,
+              secondaryOpacity: 1
+            }}
+            className={classes.title}
+            variant="h5"
+          >
+            {title}
+          </TypographyLoader>
+        </Grid>
+        <Grid
+          container
+          justify="center"
+          className={classes.content}
+          style={{ width: content.width, height: content.height }}
+        >
+          <BlockLoader loading={loading}>{children[1]}</BlockLoader>
+        </Grid>
+        <TypographyLoader
+          loading={loading}
+          loader={{
+            primaryOpacity: 0.5,
+            secondaryOpacity: 1
+          }}
+          component="span"
+        >
+          <A className={classes.sourceLink} href={sourceUrl}>
+            {sourceUrl}
+          </A>
+        </TypographyLoader>
+      </Grid>
+      <Grid container item md={4} className={classes.actionsGrid}>
+        <Grid
+          container
+          item
+          direction="row"
+          alignItems="flex-start"
+          justify="center"
+        >
+          <BlockLoader loading={loading} height="2.5rem">
+            <InsightDataActions
+              title={title}
+              onShare={handleShare}
+              onDownload={handleDownload}
+              onShowData={handleShowData}
+              onCompare={handleComapre}
+            />
+          </BlockLoader>
+        </Grid>
+        <Grid container item className={classes.summaryGrid}>
+          <BlockLoader loading={loading}>
+            <Typography className={classes.summaryHead}>
+              {summaryHead}
+            </Typography>
+            <Typography className={classes.summaryBrief}>
+              {summaryBrief}
+            </Typography>
+          </BlockLoader>
+        </Grid>
+        <Grid container item>
+          <BlockLoader loading={loading}>
+            <div style={{ width: '100%', padding: '0 1.2rem' }}>
+              <A className={classes.analysisLink} href={analysisUrl}>
+                {analysis}
+              </A>
+            </div>
+          </BlockLoader>
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+}
+
+InsightContainer.propTypes = {
+  children: PropTypes.arrayOf(PropTypes.node).isRequired,
+  title: PropTypes.string.isRequired,
+  sourceUrl: PropTypes.string,
+  loading: PropTypes.bool,
+  analysis: PropTypes.string,
+  analysisUrl: PropTypes.string,
+  summaryHead: PropTypes.string,
+  summaryBrief: PropTypes.string,
+  content: PropTypes.shape({
+    width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    height: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+  })
+};
+
+InsightContainer.defaultProps = {
+  sourceUrl: undefined,
+  loading: false,
+  analysisUrl: '/profiles/nigeria',
+  analysis: 'Read the country analysis',
+  summaryHead: 'Summary',
+  summaryBrief:
```

Comment:
Lets just call this summary.
The summaryHead  -> summaryTitle.

In most case we won't touch summaryHead so we will just set summary="Lorem ....."